person.rb

```diff
@@ -1,5 +1,8 @@
-class Person
+require './nameable'
+
+class Person < nameable
```

Comment:
- Let's capitalize `nameable` class here(`Nameable`).

Reply:
Thank you @uwadonat for the excellent suggestions. I've implemented it.